server/log_rpc_server_test.go

```diff
@@ -271,12 +272,19 @@ func TestQueueLeaves(t *testing.T) {
 	mockRegistry.EXPECT().GetLogStorage().Return(mockStorage, nil)
 	server := NewTrillianLogRPCServer(mockRegistry, fakeTimeSource)
 
-	if _, err := server.QueueLeaves(context.Background(), &queueRequest0); err != nil {
+	rsp, err := server.QueueLeaves(context.Background(), &queueRequest0)
+	if err != nil {
 		t.Fatalf("Failed to queue leaf: %v", err)
 	}
+	if len(rsp.QueuedLeaves) != 1 {
+		t.Errorf("QueueLeaves() returns %d leaves; want 1", len(rsp.QueuedLeaves))
+	}
+	if !reflect.DeepEqual(queueRequest0.Leaves[0], rsp.QueuedLeaves[0].Leaf) {
+		t.Errorf("QueueLeaves()=%+v,nil; want %+v,nil", rsp.QueuedLeaves, queueRequest0.Leaves)
+	}
 }
 
```

Comment:
Checked `OK` and added a repeat arm that checks for `ALREADY_EXISTS`.

(Aside: I'd love to convert this file to table-driven tests, but it's so far beyond the scope of this PR that I'm not even going to add a TODO.  At least, not one with my name attached to it...)